docs/guides/cloud/integrations/github-integration.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: GitHub Integration
-sidebar_position: 70
+sidebar_position: 20
+sidebar_label: GitHub
```

Comment:
for better URLs, we should either rename the md files to take out the '-integration' (so we don't repeat the word integration in the URL), or use the slug meta data item to specify a different URL.